### PR TITLE
Proxistore: gdpr compliant

### PR DESCRIPTION
**Pull request by vincentproxistore**
## Type of change
- [ ] Bugfix
- [ X] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [ ] Other

## Description of change
Make the adapter gdpr compliant

-vincent.descamps@proxisotre.com

### CODE REVIEW

File: config.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine. See: https://circleci.com/docs/2.0/configuration-reference
```
Comment (idettman) [inline config.yml]: The changes to `config.yml` should be broken out into a separate pull request, everything else looks good

**Comment by bretg** [comment]
@vincentproxistore - what's the difference between this and https://github.com/prebid/Prebid.js/pull/4527 ?
